worldengine/cli/main.py

```diff
@@ -317,6 +320,9 @@ def main():
     g_generate.add_argument('--scatter', dest='scatter_plot',
                             action="store_true", help="generate scatter plot")
 
+    g_generate.add_argument('--sat', dest='satelite_map',
```

Comment:
I wouldn't mind making this a default output. It gives a beautiful overview over the world.
What do you think, @ftomassetti and @psi29a ?